client/components/data/transaction/README.md

```diff
@@ -0,0 +1,28 @@
+CartData
```

Comment:
Oops, I'm not using this module anymore but created a module to handle both a temporary cart and the transaction store here `client/my-sites/upgrades/trials/data/free-trial-cart.jsx`.
So I'm going to remove `client/components/data/transaction`.